docker/cygnus-ngsi-ld/Dockerfile

```
@@ -346,3 +346,13 @@ ENTRYPOINT ["/cygnus-entrypoint.sh"]
 # Ports used by cygnus-ngsi
 EXPOSE ${CYGNUS_SERVICE_PORT} ${CYGNUS_API_PORT}
 
+LABEL "maintainer"="Cygnus Team. Telefónica I+D"
+LABEL "org.opencontainers.image.authors"="iot_support@tid.es"
+LABEL "org.opencontainers.image.documentation"="https://fiware-cygnus.rtfd.io/"
+LABEL "org.opencontainers.image.vendor"="Telefónica Investigación y Desarrollo, S.A.U"
+LABEL "org.opencontainers.image.licenses"="AGPL-3.0-only"
+LABEL "org.opencontainers.image.title"="Cygnus-LD Data Peristence"
```

```suggestion
LABEL "org.opencontainers.image.title"="Cygnus-LD Data Persistence"
```

Similar typo in other LABEL in this PR

---

Typo Fixed 34b9ea2